test/smoke/functions.sh

```diff
@@ -0,0 +1,70 @@
+# This file used for import in other files
+
+RED='\033[0;31m'
+NC='\033[0m'
+
+function check_mandatory_vars() {
+
+    declare -a mandatory
+    declare -a allure_mandatory
+
+    if [[ "${CI:-false}" == "true" ]]; then
+      mandatory=(
+        WALLARM_API_TOKEN
+        WALLARM_API_HOST
+        WALLARM_API_PRESET
+        CLIENT_ID
+        USER_TOKEN
+        WEBHOOK_API_KEY
+        WEBHOOK_UUID
+        PYTEST_WORKERS
+        SMOKE_REGISTRY_TOKEN
+        SMOKE_REGISTRY_SECRET
+      )
+    else
+      mandatory=(
+        WALLARM_API_TOKEN
+        WALLARM_API_HOST
+        WALLARM_API_PRESET
+        CLIENT_ID
+        USER_TOKEN
+        SMOKE_REGISTRY_TOKEN
+        SMOKE_REGISTRY_SECRET
+        WEBHOOK_API_KEY
+        WEBHOOK_UUID
+        PYTEST_WORKERS
```

Comment:
why `PYTEST_WORKERS` is required? it's not a credentials or something environment specific